src/modules/avformat/consumer_avformat.c

```diff
@@ -1896,8 +1896,26 @@ static void *consumer_thread( void *arg )
 					if ( mlt_properties_get_int( frame_properties, "rendered" ) )
 					{
 						AVFrame video_avframe;
+						int is_interlaced_chroma_correction = 0;
+
 						mlt_frame_get_image( frame, &image, &img_fmt, &img_width, &img_height, 0 );
 
+						// Interlaced 420 correction
+						if ( mlt_properties_get( frame_properties, "progressive" )
+						     && mlt_properties_get_int( frame_properties, "progressive" ) == 0
+						     && pix_fmt == AV_PIX_FMT_YUV420P  // dst
+						     && img_fmt == mlt_image_yuv422    // src. It looks like rgb and 444 go as 422 too.
```

Comment:
> It looks like rgb and 444 go as 422 too.

`mlt_consumer` has a render thread that defaults to `mlt_image_yuv422`. If you want to change it to RGB, you need to set consumer property `mlt_image_format=rgb` (or `rgba`). There is no YUV 4:4:4 in MLT.
Is it necessary to check the MLT image format (`img_fmt`) here? You are manipulating only `converted_avframe` which is based on `pix_fmt`.

Reply:
I don't want change to RGB. I just noticed side-effect, that RGB file on input works too with this fix. 
About img_fmt: I tried to check as much as possible to not to break other formats. It is just couple of assembly instructions. What if input format would be already 420. My correction would be wrong. I would prefer to do this check.

Reply:
I understand your concern, and you can instead change that line to
`img_fmt != mlt_image_yuv420p`
I let you decide; it is not a required change.

Reply:
I prefer to stay "as is" here.